.env.example

```diff
@@ -39,6 +39,7 @@ APP_URL=http://localhost:${APP_PORT}
 #INTERCEPTOR_CONNECT=
 
 #DEBUGBAR_ENABLED=false
+#VITE_REACT_QUERY_DEVTOOLS_ENABLED=false
```

Comment:
The queries and mutations on the page are complex enough that I needed to reach for the query devtools to fully wrap my brain around what's going on.

That's what this thing is at the bottom right of the screen:
![Screenshot 2024-09-22 at 9 35 35 AM](https://github.com/user-attachments/assets/c4715804-a79f-4c37-941a-19995bb54c8b)